templates/app/client/components/auth(auth)/auth.service.js

```diff
@@ -161,8 +161,10 @@ export function AuthService($location, $http, $cookies, $q, appConfig, Util, Use
     hasRole(role, callback?: Function) {
       return Auth.getCurrentUser(undefined)
         .then(user => {
-          var has = user.hasOwnProperty('role')
+          var has = user
+            ? user.hasOwnProperty('role')
             ? hasRole(user.role, role)
+            : false
```

Comment:
@Climax777 Could you please elaborate more? What is the issue? Could you provide steps to reproduce?

Thanks